src/shared/components/SkeletonPage/SkeletonPage.styles.ts

```
@@ -0,0 +1,14 @@
+import { Container, styled } from '@mui/material';
+
+export const SkeletonContainer = styled(Container)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  alignContent: 'center',
+  padding: theme.spacing(24, 4),
+  gap: theme.spacing(4),
+  textAlign: 'center',
+}));
+
+export const SkeletonPageImage = styled('img')(({}) => ({
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)